src/scripts/main.js

```diff
@@ -357,4 +357,38 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const mainTable = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in your HTML. If it doesn't, `mainTable` will be `null`, and attempting to append children to it will result in a runtime error. Consider adding a check to ensure `mainTable` is not `null` before proceeding with the loop.